src/renderer/kyc/application/current-application-container.jsx

```diff
@@ -46,11 +46,14 @@ class CurrentApplicationComponent extends PureComponent {
 		}
 		const error = requirements.reduce((acc, curr) => {
 			if (acc) return acc;
-			if (!curr.options || !curr.options.length) return true;
+			if (!curr.required) return false;
```

Comment:
should this be applied only to empty attributes?

Reply:
You are correct!
Pushed an update